src/registry_schemas/schemas/conversion.json

```diff
@@ -8,46 +8,53 @@
   "title": "Conversion Filing",
   "properties": {
     "conversion": {
-      "required": [
-        "nameRequest",
-        "offices",
-        "parties",
-        "shareStructure",
-        "contactPoint"
-      ],
       "type": "object",
-      "title": "Conversion Filing",
-      "properties": {
-        "nameRequest": {
-          "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/name_request"
-        },
-        "offices": {
-          "registeredOffice": {
-            "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/office"
-          },
-          "recordsOffice": {
-            "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/office"
-          },
-          "required": [
-            "registeredOffice"
-          ]
-        },
-        "parties": {
-          "type": "array",
-          "items": {
-            "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/parties#/definitions/party"
+      "required": [
+          "offices",
+          "parties",
+          "contactPoint"
+        ],
+        "properties": {
+          "business": {
+            "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/business#/properties/business"
+          },
+          "startDate": {
+            "type": "string",
+            "format": "date"
+          },
+          "nameRequest": {
+            "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/name_request"
+          },
+          "offices": {
+            "registeredOffice": {
+              "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/office"
+            },
+            "recordsOffice": {
+              "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/office"
+            },
+            "businessOffice": {
+              "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/office"
```

Comment:
Wrong schema for all 3 addresses: should be https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/address.json

Reply:
Please add some unit tests for the different possible offices.

Reply:
> Wrong schema for all 3 addresses: should be https://github.com/bcgov/business-schemas/blob/main/src/registry_schemas/schemas/address.json

Actually, I think businessOffice contains registeredOffice + recordsOffice so it is, in fact, of type "office".

But I'm pretty sure the first 2 are single addresses.

Reply:
Forgive me! You had this right. (I was comparing with the UI code and it is the one that is wrong.)

This is all good, no changed needed (except maybe adding some unit tests).